bundles/pixi.js-node/src/adapter/polyfills.ts

```diff
@@ -1,6 +1,52 @@
-import fetch from 'cross-fetch';
+import { fetch, Response, Request } from 'cross-fetch';
 
-globalThis.fetch = fetch;
+import fs from 'fs';
+import path from 'path';
+
+// eslint-disable-next-line func-names
+globalThis.fetch = function (url, options)
```

Comment:
In Node.js 18, native `fetch` will be standard. Setting `globalThis.fetch` will always override the internal `fetch`, which is polluting the global space. This should probably be a [ponyfill](https://github.com/sindresorhus/ponyfill) instead. 

Node's fetch will not support local paths, so this is a non-standard implementation of fetch, therefore it should probably be something else.

Reply:
Maybe we need an adapter entry for fetch?

Reply:
added!